src/vss_tools/vspec/vssexporters/utils.py

```diff
@@ -53,6 +53,11 @@ def get_trees(
 
     if extended_attributes:
         VSSNode.whitelisted_extended_attributes = list(extended_attributes)
+        for item in VSSNode.whitelisted_extended_attributes:
```

Comment:
The better place would be to do that directly in the [cli_options.py](https://github.com/COVESA/vss-tools/blob/master/src/vss_tools/vspec/cli_options.py) where the argument is defined.

We can write a function and define it for the `type=`:

```python
from click import option

def no_comma_type(value):
    if "," in value:
        raise click.BadParameter(f"'{value}' should not contain ','")

extend_all_attributes_opt = option(
    "--extend-all-attributes/--no-extend-all-attributes",
    help="""
        Generates all extended attributes,
        not only the ones given via '-e/--extended-attributes'
    """,
    default=False,
    show_default=True,
    type=no_comma_type
)
```